exercises/resistor-color-duo/README.md

```diff
@@ -0,0 +1,52 @@
+# Resistor Colors
```

Comment:
Don't forget to rename this too!

```suggestion
# Resistor Color Duo
```

Reply:
Yes, as soon as the updates to `problem-specifications` are merged in I will be running `configlet generate` to recreate the `README.md` file along with testing the generator for the spec file with the new `canonical-data.json`.